Action.swift

```diff
@@ -18,6 +18,11 @@ public final class Action<Input, Element> {
     public let _enabledIf: Observable<Bool>
     public let workFactory: WorkFactory
 
+    /// Inputs that execute the action.
+    /// Inputs via execute() also appear in this subject.
+    public let inputs = PublishSubject<Input>()
```

Comment:
What should happen if the action is not enabled but someone sends a value on this subject?